src/AWSSQS.jl

```diff
@@ -345,7 +355,7 @@ Approximate number of messages not visible in a queue.
 
 function sqs_busy_count(queue::AWSQueue)
 
-    parase(Int,sqs_get_queue_attributes(queue)["ApproximateNumberOfMessagesNotVisible"])
+    parse(Int,sqs_get_queue_attributes(queue)["ApproximateNumberOfMessagesNotVisible"])
```

Comment:
Thanks @jademackay ! I've committed this change here: https://github.com/samoconnor/AWSSQS.jl/commit/75f18e73360b0e62f056b4610b534a4a747c4f75